scalameta/scalameta/src/test/scala/scala/meta/tests/internal/LoggerTest.scala

```diff
@@ -0,0 +1,14 @@
+package scala.meta.tests.internal
+
+import org.scalameta.internal.FileLine
+
+class LoggerTest extends org.scalatest.FunSuite {
+  test("basic") {
+    val fileLine = FileLine.generate
+    assert(fileLine.file.value.endsWith("LoggerTest.scala"))
+    assert(fileLine.line.value == 7)
+    import org.scalameta.internal.logger
+    logger.elem(fileLine)
+    logger.info(fileLine)
```

Comment:
How do we test the behavior of these `logger` calls? Just eyeballing, or I'm missing something?